src/commonMain/kotlin/exchange.dydx.abacus/processor/router/skip/SkipProcessor.kt

```diff
@@ -215,10 +216,21 @@ internal class SkipProcessor(
                     token["skipDenom"] = denom
                     token["denom"] = NATIVE_TOKEN_DEFAULT_ADDRESS
                 }
-                filteredTokens.add(token.toMap())
+                tokensWithSkipDenom.add(token.toMap())
             }
         }
-        return filteredTokens
+
+        /*
+            Context: https://dydx-team.slack.com/archives/C0738MMRX9C/p1722978579167729?thread_ts=1722974628.149899&cid=C0738MMRX9C
+            We are hitting ibc rate limits for USDT https://ibc.range.org/ibc/rate-limits
+            As a result, users' funds are getting stuck mid transfers for USDT Axelar swaps into cosmos.
+            To remediate this, skip has limited high value transfers while we are hiding the route from the UI
+         */
+        return tokensWithSkipDenom.filter {
+            val token = parser.asMap(it)
```

Comment:
Nit: Would to nice to add a unit test case.

Reply:
Oh.. it's temporary.. Maybe in the future we can add a feature flag (or a config entry) that contains a list of tokens to be filtered out.

Reply:
that's a great idea... here's the ticket! https://linear.app/dydx/issue/OTE-670/[investigate]-enable-dynamic-config-to-filter-tokens